src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -934,7 +934,9 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                     Assert.Equal(HttpStatusCode.OK, response.StatusCode);
                     TestHelper.VerifyRequestMethod(response, method);
                     string responseContent = await response.Content.ReadAsStringAsync();
-                    _output.WriteLine(responseContent);                    
+                    _output.WriteLine(responseContent);
+
+                    Assert.Contains($"\"Content-Length\": \"{request.Content.Headers.ContentLength.Value}\"", responseContent);
```

Comment:
It's to verify that we're appropriately sending Content-Length headers... there is no other test in the repo doing so.  I went to add such a test for all of the verbs that can upload, and then I saw we already had such a test and just weren't checking whether Content-Length was being sent, so I added it here (the test would have been basically duplicated entirely otherwise).  We previously had a bug on unix where Content-Length wasn't being sent when needed, so I wanted a test to catch a future regression.